### PR TITLE
Update babel and other dependencies

### PR DESCRIPTION
**Pull request by jpeyper**
Add deprecation message

Update dependencies to fix security vulnerabilities.

### CODE REVIEW

File: packages/react-redux-dynamic-reducer/.babelrc [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
-  "presets": ["env", "react", "stage-3"]
+  "presets": [
+    "@babel/preset-env",
+    "@babel/preset-react"
+  ],
+  "plugins": [
+    "@babel/plugin-syntax-dynamic-import",
+    "@babel/plugin-syntax-import-meta",
+    "@babel/plugin-proposal-class-properties",
+    "@babel/plugin-proposal-json-strings"
```
Comment (mpeyper) [inline packages/react-redux-dynamic-reducer/.babelrc]: what does `plugin-syntax-import-meta` and `plugin-proposal-json-strings` do?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # redux-dynamic-reducer
 
+## Deprecated
+
+Our new redux library, [redux-dynostore](https://github.com/ioof-holdings/redux-dynostore) has all the features of this one and more, allowing for a lot more dynamic store features. If you experience any difficulty switching over to [redux-dynostore](https://github.com/ioof-holdings/redux-dynostore) then please let us know by raising an issue over there. This library will be subject to major bug fixes and security fixes only.
```
Comment (mpeyper) [inline README.md]: > all the features of this one and more

Are we sure?

**Review by mpeyper — APPROVED** [review]

File: packages/react-redux-dynamic-reducer/.babelrc (hunk 1 above)
Comment (jpeyper) [reply packages/react-redux-dynamic-reducer/.babelrc]: I used [babel-upgrade](https://github.com/babel/babel-upgrade) to do the upgrade, I think this replaces the stage-3 preset that is no longer supported.

File: README.md (hunk 2 above)
Comment (jpeyper) [reply README.md]: Nope, but if its missing some I want other people to help us find them... all the major features are (to my knowledge)

**Pull request merged**